modules/command-engine/javadsl/src/main/scala/surge/javadsl/common/Context.scala

```diff
@@ -0,0 +1,13 @@
+// Copyright © 2017-2021 UKG Inc. <https://www.ukg.com>
+
+package surge.javadsl.common
+
+import surge.core.{ Context => Core }
+
+trait Context {}
+
+object Context {
+  def apply(core: Core): Context = new ContextImpl(core)
+}
+
+class ContextImpl(private val core: Core) extends Context
```

Comment:
Minor but I think `ContextImpl` can be internal and we can keep the `Context` trait as the public/backwards compatible interface